kubernetes-discovery-client/src/main/java/io/micronaut/kubernetes/client/v1/KubernetesServiceConfiguration.java

```diff
@@ -36,16 +36,26 @@
     private String serviceId;
     private String name;
     private String namespace;
+    private String mode;
+    private String port;
+    private final boolean manual;
 
     @Inject
     public KubernetesServiceConfiguration(@Parameter String serviceId) {
-        this.serviceId = serviceId;
+        this(serviceId, null, null, null, null, true);
+    }
+
+    public KubernetesServiceConfiguration(String serviceId, String name, String namespace, boolean manual) {
+        this(serviceId, name, namespace, null, null, manual);
     }
 
-    public KubernetesServiceConfiguration(String serviceId, String name, String namespace) {
+    public KubernetesServiceConfiguration(String serviceId, String name, String namespace, String mode, String port, boolean manual) {
```

Comment:
This is a breaking change. Can the existing constructor be left as is, and perhaps you add a new one?